packages/target-electron/src/application-config.ts

```diff
@@ -1,11 +1,17 @@
 // @ts-ignore
 import applicationConfig from 'application-config'
+if (process.env.NODE_ENV !== 'production') {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const { config } = require('dotenv')
```

Comment:
its in devDependencies anyway, so it won't be installed in production

Reply:
> its in devDependencies anyway, so it won't be installed in production

that's not how it works here. esbuild bundles it. but I wonder more why require is even allowed here because this is an esm module, I thought there was some workaround to required to get the original `require`

so there is still the tree shake questions remaining, is this still in the bundle in production builds? afaik only the frontend has `NODE_ENV=production` with `pnpm run build4production` or have I missed something.

I would just make this dependent on `--devmode` cli option unless we really have a good workflow that excludes it in production.

Reply:
we can convert this into an issue and merge this already for the test release

Reply:
I just tried await import('dotenv') instead and it seems to work: found that here: https://dmitripavlutin.com/ecmascript-modules-dynamic-import/

I will test again in production mode

Reply:
It works as expected. Even if I uninstall dotenv I can build for production. Added a try catch to make sure it won't influence prod build